src/components/App/App.js

```diff
@@ -29,6 +26,7 @@ class App extends Component {
   findMovie = (id) => {
       this.setState({movieID: id})
       this.hanldeSingleMovie(id)
+      //this is where route id is defined?
     }
     
   hanldeSingleMovie = (id) => {
```

Comment:
Probably a good idea to do more with the Error in the singleMovie fetch than just console.log it - might be a good idea to set it in state